sklearn/datasets/tests/test_openml.py

```diff
@@ -503,6 +505,18 @@ def test_fetch_openml_as_frame_auto(monkeypatch):
     assert isinstance(data.data, scipy.sparse.csr_matrix)
 
 
+def test_fetch_openml_default_as_frame_warning(monkeypatch):
+    data_id = 61  # iris dataset version 1
+    _monkey_patch_webbased_functions(monkeypatch, data_id, True)
+    assert_warns_message(
```

Comment:
Could you use pytest instead with something along the lines:

```python
with pytest.warns(RuntimeWarning) as record:
    warnings.warn("another warning", RuntimeWarning)

# check that only one warning was raised
assert len(record) == 1
# check that the message matches
assert record[0].message.args[0] == "another warning"
```

Reply:
test updated